charts/buildkit-service/templates/deployment.yaml

```
@@ -93,6 +93,10 @@ spec:
           {{- end }}
           resources:
             {{- toYaml .Values.resources | nindent 12 }}
+          {{- with .Values.lifecycle }
```

This here is missing a bracket which made the lint job fail.

```suggestion
          {{- with .Values.lifecycle }}
```

---

Oops, thanks for catching this! Fixed via 564ef14